app/jobs/judges/CF.rb

```diff
@@ -80,7 +72,25 @@ def fetch_submission_status
     last_submission_row = page.search('.datatable tr')[1]
     submission_details = last_submission_row.search('td').map(&:text).map(&:strip)
     keys = %w(run_id username problem_name language verdict time memory send_time judged_time favorite compare)
-    Hash[keys.zip(submission_details)]
+    status = Hash[keys.zip(submission_details)]
+    verdict = status['verdict']
+    return nil if verdict.nil? || verdict.empty? || verdict.include?("ing")
+    {
+      verdict: format_verdict(verdict),
+      time: verdict.to_i,
+      memory: verdict.to_i,
+    }
+  end
+
+  def get_status_dict(response)
+    status = DETAIL_REGEX.match(response).named_captures
+    verdict = status['verdict']
+    return nil if verdict.nil? || verdict.empty? || verdict.include?("ing")
+    {
+      verdict: format_verdict(verdict),
+      time: verdict.to_i,
```

Comment:
Is it intended to not give the time and memory here?

Reply:
No, just a typo :)